src/Exception/InvalidPhoneNumberException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber\Exception;
+
+use InvalidArgumentException;
+
+use function sprintf;
+
+final class InvalidPhoneNumberException extends InvalidArgumentException implements ExceptionInterface
+{
+    public static function with(string $phoneNumber, ?string $countryCode): self
+    {
+        if (! empty($countryCode)) {
```

Comment:
Let's not use `empty()`, please (like ever :D )

`empty($v)` is equivalent to `! isset($v) || $v === '' || $v === false || $v === 0 || $v === '0' || $v === [] || $v === 0.0` (and more stuff perhaps - can't even remember).

Never use it, if possible.